src/Files.App/Helpers/Application/AppLifecycleHelper.cs

```diff
@@ -317,7 +317,7 @@ public static void HandleAppUnhandledException(Exception? ex, bool showToastNoti
 				{
 					await Launcher.LaunchUriAsync(new Uri("files-uwp:"));
 				})
-				.Wait(1000);
+				.Wait(100);
```

Comment:
This is not directly related to fixing the crash, but it does make it easier for an automatic restart to succeed in the event of a crash.

Reply:
Thanks! I noticed that it hasn't been restarting that often.